tests/vppoutputencode.cpp

```diff
@@ -103,10 +107,15 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encVideoParams.intraPeriod = encParam->intraPeriod;
     encVideoParams.ipPeriod = encParam->ipPeriod;
     encVideoParams.rcParams.bitRate = encParam->bitRate;
+    encVideoParams.rcParams.targetPercentage = encParam->targetPercentage;
```

Comment:
Yes, libyami should make sure it work well, no matter caller set the value or not